ide/refactoring.api/src/org/netbeans/modules/refactoring/spi/impl/RefactoringPanel.java

```diff
@@ -179,6 +181,11 @@ private void initialize() {
         add(splitPane, BorderLayout.CENTER);
         splitPane.setRightComponent(new JLabel(org.openide.util.NbBundle.getMessage(RefactoringPanel.class, "LBL_Preview_not_Available"), SwingConstants.CENTER));
         splitPane.setBorder(null);
+        splitPane.addPropertyChangeListener(JSplitPane.DIVIDER_LOCATION_PROPERTY, evt -> {
+            if (evt.getNewValue() instanceof Integer pos && pos > MIN_DIVIDER_LOCATION) {
+                RefactoringPanel.dividerPosMemory = pos;
+            }
+        });
```

Comment:
Would it make sense to add the "minium width" logic also to `platform/api.search/src/org/netbeans/modules/search/ui/BasicSearchResultsPanel.java` and `platform/api.search/src/org/netbeans/modules/search/ui/BasicReplaceResultsPanel.java`? If I'm not mistaken there you can set the divider location to complete left and it will stay there.

Reply:
Ignore this, covered in #8088